RecoTracker/FinalTrackSelectors/plugins/MultiTrackSelector.cc

```diff
@@ -561,10 +575,13 @@ void MultiTrackSelector::processMVA(edm::Event& evt, const edm::EventSetup& es,
     return;
   }
 
+  if(!useMVA_[selIndex] && !writeIt)return;
```

Comment:
If the MVA is not used in this selector, there's no need to calculate the MVA, so the function bails out. The two reasons the MVA is used in the actual selection or to be used for saving in the ValueMap, so this AND evaluates to false if we're not doing either.